client/src/modules/app/AppRebuild.tsx

```diff
@@ -69,7 +69,7 @@ export const AppRebuild = ({ appId }: AppRebuildProps) => {
       });
       setIsTerminalVisible(true);
       setRebuildLoading(true);
-    } catch (e) {
+    } catch (e: any) {
```

Comment:
could you please add the flag to disable this ts option in the tsconfig file instead ?

Reply:
:heavy_check_mark: Added `useUnknownInCatchVariables` as `false` + reverted `any` error casts in commits:
- https://github.com/ledokku/ledokku/pull/421/commits/45e6af0b1e775106e51c3aee05878cd102b3ec23
- https://github.com/ledokku/ledokku/pull/421/commits/ffb29378c48f4056c78d709b5ee64a5008b03d39